lib/src/header_parser/clang_bindings/clang_bindings.dart

```diff
@@ -1706,6 +1837,24 @@ class CXChildVisitResult {
   static const int CXChildVisit_Recurse = 2;
 }
 
+class CXEvalResultKind {
```

Comment:
Generate a comment that this is from an enum, and that the actual enum values are exposed in Dart as integers for functions and struct fields.

Reply:
I am not really sure what you mean.
This enum is part of libclang itself, not our wrapper, so we cannot add a comment to a particular enum.

Reply:
Ah, yeah of course. I was thinking because this enum value did not have a doc-comment copied from C we could generate explanatory comments. But having both a doc-comment and a generated comment is going to be very confusing for people.

Then just leave it as is.